Documentation/RVEngine/Host/EXTENSIONS.md

```
@@ -0,0 +1,105 @@
+DLLEXPORT void CDECL pre_start();
+DLLEXPORT void CDECL post_start();
+DLLEXPORT void CDECL pre_init();
+DLLEXPORT void CDECL post_init();
+DLLEXPORT void CDECL on_frame();
+DLLEXPORT void CDECL mission_ended();
+// ... и другие
+```
+
+## Структура папок
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)